patches/0003-glue.cc-Change-__assert_fail-function-signature.patch

```diff
@@ -0,0 +1,29 @@
+From 33cee463dd74661440551b676104107df07a193b Mon Sep 17 00:00:00 2001
+From: Job Paardekooper <jobpaardekooper@users.noreply.github.com>
```

Comment:
This should be your name, @StefanJum , right?

Reply:
Done